lab_5_scraper/scraper_config.json

```diff
@@ -1,9 +1,36 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://gtrksakha.ru/news/",
+        "https://gtrksakha.ru/",
+        "https://gtrksakha.ru/rubrics/society/",
+        "https://gtrksakha.ru/rubrics/culture/",
+        "https://gtrksakha.ru/rubrics/sports/"
+    ],
+    "headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"},
+    "total_articles_to_find_and_parse": 100,
+    "encoding": "utf-8",
+    "timeout": 50,
     "should_verify_certificate": true,
-    "headless_mode": true
+    "headless_mode": true,
+    "seed_urls_all_topics": ["https://gtrksakha.ru/news/",
+        "https://gtrksakha.ru/",
+        "https://gtrksakha.ru/rubrics/society/",
+        "https://gtrksakha.ru/rubrics/culture/",
+        "https://gtrksakha.ru/rubrics/sports/",
+        "https://gtrksakha.ru/rubrics/education/",
+        "https://gtrksakha.ru/rubrics/health/",
+        "https://gtrksakha.ru/rubrics/politics/",
+        "https://gtrksakha.ru/rubrics/transport/",
+        "https://gtrksakha.ru/rubrics/it/"
+        ],
+    "seed_urls_100": ["https://gtrksakha.ru/news/",
+                  "https://gtrksakha.ru/",
+                  "https://gtrksakha.ru/rubrics/society/",
+                  "https://gtrksakha.ru/rubrics/culture/",
+                  "https://gtrksakha.ru/rubrics/sports/"],
+    "seed_urls_150": ["https://gtrksakha.ru/news/",
+                  "https://gtrksakha.ru/",
+                  "https://gtrksakha.ru/rubrics/society/",
+                  "https://gtrksakha.ru/rubrics/culture/",
+                  "https://gtrksakha.ru/rubrics/sports/",
+                  "https://gtrksakha.ru/rubrics/education/"]
```

Comment:
As we discussed previously, you just need one pack with all seed_urls. Number of articles for collection is not defined by the seed_urls size, but the total_articles_to_find_and_parse parameter.